bots/naughty/fedora-30/11614-selinux-denying-iscsi_tcp-module-load

```diff
@@ -0,0 +1,3 @@
+Traceback (most recent call last):
+  File "/build/cockpit/bots/../test/verify/check-machines-dbus", line *, in testStoragePoolsCreate
```

Comment:
The file path can be specified as `test/verify/check-machines-dbus`.  Also maybe the naughty would be a bit safer if you would specify that the dialog also fails to close, but I think it is just fine :)
Now fails only testCPUSecurityMitigations which is regression outside of this PR.